core/io/packet_peer.cpp

```diff
@@ -102,7 +102,7 @@ Error PacketPeer::put_var(const Variant &p_packet, bool p_full_objects) {
 	ERR_FAIL_COND_V_MSG(len > encode_buffer_max_size, ERR_OUT_OF_MEMORY, "Failed to encode variant, encode size is bigger then encode_buffer_max_size. Consider raising it via 'set_encode_buffer_max_size'.");
 
 	if (unlikely(encode_buffer.size() < len)) {
-		encode_buffer.resize(0); // Avoid realloc
+		encode_buffer.clear(); // Avoid realloc
```

Comment:
```suggestion
		encode_buffer.clear(); // Avoid realloc.
```

Reply:
Note that style feedback should normally be done after regular review, to not add work to contributors before the changes have been approved